docs/development/getting_started.rst

```diff
@@ -1,6 +1,14 @@
 Getting Started
 ===============
 
+.. note:: SecureDrop maintains two branches of documentation, ``stable`` and
+          ``latest``, the former being the default on our Read the Docs powered
+          site. ``stable`` is built from our latest signed git tag, while
+          ``latest`` is built from the head of the ``development`` git branch.
```

Comment:
This branch is named `develop`, not `development`. While you might reasonably call it "the development branch" to describe it in the context of some of the other branches in the upstream repository, the wording of this sentence makes it sound like that is also the name of the branch, which is potentially confusing.